products/lua.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Lua
+category: lang
+iconSlug: lua
+permalink: /lua
+versionCommand: |-
+  lua -v
+releaseImage:
+  https://www.lua.org/images/timeline.png
+releasePolicyLink: https://www.lua.org/versions.html
+changelogTemplate: |
+  https://www.lua.org/versions.html#{{"__RELEASE_CYCLE__"| split: " " | first}}/
```

Comment:
```suggestion
changelogTemplate: https://www.lua.org/versions.html#{{"__RELEASE_CYCLE__"| split: " " | first}}/
```

Formatting.

Reply:
broke CI:

```
:24:34 PM: Submodule path "_data/release-data": checked out "62e1c4c0a969b8d6caba1b4d91a1d22f642a3483"
5:24:34 PM: Traceback (most recent call last):
5:24:34 PM:   File "_data/release-data/latest.py", line 24, in <module>
5:24:34 PM:     class ReleaseCycle:
5:24:34 PM:   File "_data/release-data/latest.py", line 38, in ReleaseCycle
5:24:34 PM:     def latest(self) -> str | None:
5:24:34 PM: TypeError: unsupported operand type(s) for |: "type" and "NoneType"
5:24:34 PM: Replacing Deploy URL with Preview URL (https://deploy-preview-4433--endoflife-date.netlify.app/)
5:24:34 PM: Building site...
5:24:35 PM: Configuration file: /opt/build/repo/_config.yml
5:24:36 PM: Failed during stage "building site": Build script returned non-zero exit code: 2
5:24:36 PM: bundler: failed to load command: jekyll (/opt/build/cache/bundle/ruby/3.1.0/bin/jekyll)
5:24:36 PM: /opt/buildhome/.rvm/rubies/ruby-3.1.4/lib/ruby/3.1.0/psych.rb:455:in `parse": (products/lua.md): mapping values are not allowed in this context at line 9 column 82 (Psych::SyntaxError)
	from /opt/buildhome/.rvm/rubies/ruby-3.1.4/lib/ruby/3.1.0/psych.rb:455:in `parse_stream"
	from /opt/buildhome/.rvm/rubies/ruby-3.1.4/lib/ruby/3.1.0/psych.rb:399:in `parse"
```

https://app.netlify.com/sites/endoflife-date/deploys/65a3ee8229b8990008c09129

so reverted

Reply:
Fixed in 9942ca7c81c362fe4f962f14410b28d178c72f0e.